src/main.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(btree_drain_filter)]
```

Comment:
As stated in my comment on the issue https://github.com/Rahix/atdf2svd/issues/13#issuecomment-732113470, this would prevent building atdf2svd on stable rust which is something I'd very much like to retain.  In the comment, I've hinted at a possible alternative solution, can you try that?

Reply:
Done. Than you for clarification and sorry for lack of knowledge. I'm learning Rust and MCU in same time. I really appreciate collaboration with you @Rahix. Thanks one more time :+1:

Reply:
Nothing to be sorry about, I'm happy to help where I can!  And the appreciation is mutual, I enjoy working with you just as much, it's been a pleasure and you're doing a ton of great work!